mkdocs.yml

```diff
@@ -33,7 +33,7 @@ pages:
 - Messaging: messaging.md
 - Storage: storage.md
 - LoRa: lora.md
-- TLS: tls.md
+- Securing communication: secure-communication.md
```

Comment:
Maybe you should rename this chapter in `Security`, and `Communication` should be only one subchapter here. Chapter can hold other topics on Mainflux security later.